File: clientandroid/app/src/main/java/com/hairdresser/pyshankov/fragment/SearchFragment.java
package com.hairdresser.pyshankov.fragment;

import android.content.Context;
import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.hairdresser.pyshankov.hairdresser.R;

/**
 * Created by pyshankov on 1/29/17.
 */

public class SearchFragment extends NamedFragment {

    public static final String NAME = "Search";


    public SearchFragment() {
        this.name = NAME;
    }


    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        return inflater.inflate(R.layout.fragment_search, container, false);
    }


    @Override
    public void onAttach(Context context) {
        super.onAttach(context);
    }

    @Override
    public void onDetach() {
        super.onDetach();
    }


}

File: clientandroid/app/src/main/kotlin/com/hairdresser/pyshankov/fragment/SearchFragment.kt
package com.hairdresser.pyshankov.fragment

import android.content.Context
import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

import com.hairdresser.pyshankov.hairdresser.R

/**
 * Created by pyshankov on 1/29/17.
 */

class SearchFragment : NamedFragment() {
    init {
        this.name = NAME
    }


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
    }

    override fun onCreateView(inflater: LayoutInflater?, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {
        // Inflate the layout for this fragment
        return inflater!!.inflate(R.layout.fragment_search, container, false)
    }


    override fun onAttach(context: Context?) {
        super.onAttach(context)
    }

    override fun onDetach() {
        super.onDetach()
    }

    companion object {

        val NAME = "Search"
    }


}
